explorer/data/prelude.carbon

```diff
@@ -693,31 +712,17 @@ class Optional(T:! type) {
         return x;
       }
     }
-    // TODO: Use assert as soon as available
+    Assert(false, "Attempted to unwrap empty Optional");
+    // TODO: Drop uninitialized variable & return when we can flag unreachable paths.
```

Comment:
Can the `var y: T;` be moved here, rather than having it above the `match`? (maybe there's an uninitialized state catch if you do that, but I wouldn't think so?)

Reply:
Yeah that's no problem, I just kept it there to keep the changeset minimal, but happy to move it down.